src/Parser/Core/Modules/DamageMitigation/Reductions.js

```diff
@@ -0,0 +1,449 @@
+import SPELLS from 'common/SPELLS';
+import MAGIC_SCHOOLS from 'common/MAGIC_SCHOOLS';
+import SPECS from 'Game/SPECS';
+import RACES from 'Game/RACES';
+
+import AOE_TYPE from './AOE_TYPE';
+
```

Comment:
I'd prefer it if this file didn't include spells that are class/spec specific. I believe core shouldn't know about class/spec specific things unless they can be applied to others (e.g. externals).

Reply:
Agreed, dampen harm and the fully absorbed block should be moved to a class specific extension.